plugins/convertPathData.js

```diff
@@ -772,10 +772,7 @@ function roundData(data) {
         return data;
     }
 
-    roundData = precision > 0 ? strongRound : round; // jshint ignore: line
-
-    return roundData(data);
-
+    return (precision > 0 ? strongRound : round)(data);
```

Comment:
This method will round a set of values, iterating them and invoking `Number.toFixed` in every case (and sometimes more logic like `Math.abs`). Compared to that, a single integer comparison more or less is just a micro-optimization. In my opinion, overwriting the `roundData` function reference for the sake of gaining a couple of CPU cycles is just not worth it.